package.json

```diff
@@ -14,7 +14,7 @@
     "changeset": "changeset",
     "changeset:version": "changeset version && npm install && npm run update-unpkg-links",
     "changeset:publish": "npm run build && changeset publish",
-    "create-task": "node templates/cli/src/index.js"
+    "new": "node templates/cli/src/index.js"
```

Comment:
Do we need to update the README to match the change?

Reply:
fixed